tests/test_validation.py

```diff
@@ -221,6 +221,22 @@ class Person(models.Base):
     person.name = 'Jimmy'
 
 
+def test_regex_modify_schema():
```

Comment:
There are some missing test cases:
1. Invalid flags
2. Flags provided directly in an ECMA262 pattern
3. Flags passed with `**flags`